### DISCUSSION TITLE
How can I exclude a gstreamer when installing qt

### DISCUSSION THREAD

**Discussion by ilongshan** [microsoft/vcpkg#26158] [Q&A] [answered]
I am trying to install qt with vcpkg.json file, but gstreamer has build error, and I want to exclude gstreamer when building qt. But how can I do that?
```
{
    "name": "test",
    "version-string": "0.0.1",
    "dependencies": [
        {
            "name": "qt"
        }
    ]
}
```

**Comment by dg0yt** [comment, marked as answer]
Instead of building all of `qt`, just use those ports (and maybe features) you really need. gstreamer is an opt-in feature of `qtmultimedia`, i.e. it is not activated until pulled in by you or by some port (feature) you use. (And `qt` explicitly requests `qtmultimedia[gstreamer]` for some platforms.)

**Reply by ilongshan** [reply]
Thanks for your reply. I will install them seperately. Hope there will be an easier way to handle these issues.
